src/main/resources/customdrops.yml

```diff
@@ -38,6 +38,9 @@
 # +-------------------------------------------------------------------------------------------LM3
 defaults:
   chance: 0.2
+  deathInChunkThreshold: 1000000
```

Comment:
please add temporary comments of what these options do so we can transfer it to the wiki

Reply:
(Still applies as of today's commit)

Reply:
I have added the comment?

Reply:
The other settings that were added :)